jars/urls.py

```diff
@@ -53,6 +53,7 @@
     url(r'^collection/$', views.collection_list, name="collections"),
     url(r'^collection/create/$', views.create_collection, name="create-collection"),
     url(r'^metadata/$', views.list_metadata, name='list-metadata'),
+    url(r'^collection/export/$',views.export_coauthor_data, name="export-coauthor-data"),
```

Comment:
See comment above -- consider adding the collection ID as a parameter in the route.